src/Firehose.Web/Authors/Suthahar.cs

```diff
@@ -18,7 +18,7 @@ public class Suthahar : IAmACommunityMember
 
         public IEnumerable<Uri> FeedUris
         {
-            get { yield return new Uri("https://xamarininterviewquestion.blogspot.in/feeds/posts/default"); }
+            get { yield return new Uri("https://xamarininterviewquestion.blogspot.com/feeds/posts/default"); }
```

Comment:
I assume you did this intentionally? Was the .in giving problems?

Reply:
Yeah, the actual feed returns a .com